psc-ide.el

```diff
@@ -241,6 +241,31 @@ Defaults to \"output/\" and should only be changed with
   (interactive)
   (psc-ide-send-sync psc-ide-command-quit))
 
+(defun psc-ide-server-start-with-psc-package-sources (root)
+  "start psc-ide-server with psc-package sources run in the root directory"
+  (interactive (list (read-directory-name "Project root?" (psc-ide-suggest-project-dir))))
+  (when (get-buffer "*psc-ide-server*") (kill-buffer "*psc-ide-server*"))
+  (cd root)
+  (if (file-exists-p "psc-package.json")
+      (let* ((results "*PSC-PACKAGE SOURCES*")
+             (errors "*PSC-PACKAGE ERRORS*"))
+        (shell-command "psc-package sources" results errors)
+        (if (get-buffer errors)
+            (switch-to-buffer-other-window errors)
+          (progn
+            (with-current-buffer results
+              (let* ((globs (split-string (buffer-string))))
+                (custom-set-variables
+                 '(psc-ide-source-globs
+                   (append globs
+                           (quote ("src/**/*.purs"
+                                   "bower_components/purescript-*/src/**/*.purs")))))
```

Comment:
Why would be append bower_components globs if we detected that psc-package is being used?

Reply:
I've used this a few times, but I guess maybe it's not worth having